### PR TITLE
CB-15907 Multiple Raw use of parameterized class in integ test module

### PR DESCRIPTION
**Pull request by horadla23**
See detailed description in the commit message.

### CODE REVIEW

**Review by lacikaaa — APPROVED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/distrox/DistroXTestDto.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.StackViewV4Response;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.InstanceGroupV4Response;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.instancemetadata.InstanceMetaDataV4Response;
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
```
Comment (lajosrodek) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/distrox/DistroXTestDto.java:31]: Potentially unused import.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
 import org.testng.util.Strings;
 
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
```
Comment (lajosrodek) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/environment/EnvironmentTestDto.java:21]: Potentially unused import.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/recipe/RecipeTestDto.java [hunk 3]
```diff
@@ -10,6 +10,7 @@
 import com.sequenceiq.cloudbreak.api.endpoint.v4.recipes.responses.RecipeV4Response;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.recipes.responses.RecipeViewV4Response;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.recipes.responses.RecipeViewV4Responses;
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
```
Comment (lajosrodek) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/recipe/RecipeTestDto.java:13]: Potentially unused import.

**Review by lajosrodek — APPROVED** [review]
LGTM

**Pull request merged**